src/org/traccar/database/DeviceManager.java

```diff
@@ -317,80 +317,43 @@ public void removeGroup(long groupId) throws SQLException {
         groupsById.remove(groupId);
     }
 
-    public boolean lookupServerBoolean(long deviceId, String attributeName, boolean defaultValue) {
-        String result = lookupAttribute(deviceId, attributeName, true);
+    public boolean lookupAttributeBoolean(long deviceId, String attributeName, boolean defaultValue,
+            boolean lookupServer) {
```

Comment:
I think we should change it to `lookupConfig` because server meaning is not that obvious.